libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -333,8 +350,20 @@ bool AP_Arming::battery_checks(bool report)
             }
             return false;
         }
-    }
 
+        if ((_min_voltage[0] > 0.0f) && (_battery.voltage(0) < _min_voltage[0])) {
+            if (report) {
+                GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "PreArm: Battery 0 voltage below minimum");
```

Comment:
Which is why it should be using AP_BATT_MONITOR_MAX_INSTANCES. hard coded indexes are just as bad